GVRf/Sample/gvrjassimpmodelloader/src/org/gearvrf/jassimpmodelloader/JassimpModelLoaderViewManager.java

```diff
@@ -27,7 +27,13 @@
 import org.gearvrf.animation.GVRAnimation;
 import org.gearvrf.animation.GVRAnimationEngine;
 import org.gearvrf.animation.GVRRepeatMode;
+<<<<<<< HEAD
```

Comment:
is this the result of a bad merge?  why are we seeing the <<<<<< HEAD, =======, >>>>> pattern added in this file?